content/en/_index.html

```diff
@@ -163,8 +163,9 @@
     <div class="container">
       <div class="row">
         <div class="col-xs-12 col-lg-12">
-          <div class="tag-line">Accelerate Your Continuous Delivery on Kubernetes</div>
-          <div class="tag-line-secondary mx-auto">automate the continuous delivery of change through your environments via GitOps and create previews on Pull Requests to help you accelerate</div>
+          <div class="tag-line">All In One CI/CD for Kubernetes</div>
+          <div class="tag-line-secondary mx-auto">Multi-cluster GitOps, Tekton pipelines, Pull Request previews</div>
```

Comment:
Also include secret management, I have always been a huge fan of the way jx works with secrets (using external secrets)